dev-tools/reconfigurator-cli/src/lib.rs

```diff
@@ -1185,6 +1189,8 @@ enum SetArgs {
     Seed { seed: String },
     /// target number of Nexus instances (for planning)
     NumNexus { num_nexus: u16 },
+    /// generation of the active Nexus zones (controlling handoff)
```

Comment:
I wonder if it's worth calling out explicitly somewhere around here that this is controlling the simulated state about which (simulated) Nexus instances _think_ they're active, as opposed to changing the generation in the blueprint.  ("active generation" _shouldn't_ imply a blueprint change, and obviously they're not doing `blueprint-edit` here, but I think it's still easy for someone to forget this and get confused.)